supervisord.conf

```diff
@@ -0,0 +1,18 @@
+[supervisord]
+nodaemon = true
+environment = PLACEHOLDER=true
+
+[program:etl]
+user = nodeuser
+directory = /code
+command = node schedule.js
+autorestart	= true
```

Comment:
What is the intention of the differences in spacing between these key/values? i.e. some have a single space, others have none and others have multiple spaces.